app/components/moderation-base/component.js

```diff
@@ -1,30 +1,58 @@
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import Component from '@ember/component';
+
+import { task } from 'ember-concurrency';
 /**
  * The base for the provider moderation page (tabs, provider name, and breadcrumbs).
  *
  * Sample usage:
  * ```handlebars
- * {{moderation-base
- *   provider=provider
- *   pendingCount=pendingCount
- * }}
+ * {{moderation-base}}
  * ```
  * @class moderation-base
  * */
 export default Component.extend({
-    tabs: computed('pendingCount', function() {
+    theme: service(),
+    store: service(),
+
+    tabs: computed('theme.reviewableStatusCounts.pending', function() {
         return [
             {
                 nameKey: 'global.moderation',
                 route: 'preprints.provider.moderation',
                 hasCount: true,
-                count: this.get('pendingCount') || 0,
+                count: this.get('theme.reviewableStatusCounts.pending'),
             },
             {
                 nameKey: 'global.settings',
                 route: 'preprints.provider.settings',
             },
         ];
     }),
+
+    didReceiveAttrs() {
+        this.set('providerName', this.get('theme.provider.name'));
+        this.get('fetchData').perform();
+    },
+
+    fetchData: task(function* () {
+        try {
+            const response = yield this.get('store').findRecord(
```

Comment:
This looks identical to calling `theme.loadProvider`, could we do that instead?

...but also, do we need to refresh the counts here? I think it'll usually run right after the route model hook refreshes the provider.

Reply:
Oh probably. I'll test it out

Reply:
@aaxelb, done 💰